contracts/token/TransferManager.sol

```diff
@@ -157,6 +157,12 @@ contract TransferManager is Pausable {
         revert("Transfer not possible");
     }
 
+    function batchTransfer(address[] _to, uint256[] _value) external {
```

Comment:
batchTransfer doesn't make sense I think (an individual holder would never call this method).

Reply:
Nevermind, it can, why not.

Reply:
yes, it's a request that we had from some issuers, they want to mint the total supply of tokens on their wallet and then proceed to the distribution by using the `transfer` function